cypress/e2e/journey.cy.ts

```diff
@@ -336,3 +338,22 @@ function check_page_is_project_texts() {
   cy.get('section.details');
   cy.get('.details__panel-top-text').contains('Text');
 }
+
+function checkSelectedWordIsHighlighted(){
```

Comment:
So, this is just checking that CSS rules are working? Fair enough, but I'm not so worried about that.
Is there any way we can test for the word to be scrolled into view?